assets/docker/Dockerfile.node.tmpl

```diff
@@ -1,11 +1,17 @@
 FROM node:{{ or .RuntimeVersion .DefaultRuntimeVersion }} AS build-env
-COPY . /src
-WORKDIR /src
+
+WORKDIR /app
+
+COPY package*.json tsconfig*.json ./
+
 RUN {{ .NodeInstallCommand }}
+
+COPY . /app
+
 RUN npm run build --if-present
 RUN npm test
 
 FROM gcr.io/distroless/nodejs20-debian11
-COPY --from=build-env /src /src
-WORKDIR /src
-CMD ["index.js"]
+COPY --from=build-env /app /app
+WORKDIR /app
+ENTRYPOINT ["index.js"]
```

Comment:
This will not work, the file needs to be in the CMD as reported in the README of the distroless project https://github.com/GoogleContainerTools/distroless/tree/main/nodejs

Reply:
Worked on my local.. Let me check again..

Reply:
reverted to CMD seems ENTRYPOINT is not reliable.